packages/coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -67,6 +67,15 @@ export function convertDispenseNotification(bundle: fhir.Bundle, logger: pino.Lo
   ) as fhir.IdentifierExtension
 
   const BSAId = commissionedByExtension.valueIdentifier.value
+
+  if (!BSAId){
+    throw new processingErrors.InvalidValueError(
+      // eslint-disable-next-line max-len
```

Comment:
These lines don't look that long, is this necessary?

Reply:
You are correct. Removed.